client/client_pool/include/client/client_pool/concord_client_pool.hpp

```diff
@@ -123,19 +121,19 @@ class ConcordClientPool {
                            uint64_t seq_num,
                            std::string correlation_id = {},
                            std::string span_context = std::string(),
-                           const std::function<void(SendResult&&)>& callback = nullptr);
+                           std::function<void(bftEngine::SendResult&&)> callback = nullptr);
```

Comment:
Define type for std::function<void(bftEngine::SendResult&&)>

Reply:
Will do